tictactoe/ai_minimax.py

```diff
@@ -12,15 +12,14 @@ def set_opponent_symbol(self, symbol):
 
     def move(self, board):
         if board.turn_count() == 0:
-            return 5
+            return MIDDLE_CELL
         return self.minimax(board, self._symbol)[0]
 
     def minimax(self, board, player):
         available_moves = board.empty_cells()
 
         if self.is_terminal_state(board):
-            for player in [O, X]:
-                return self.terminal_state_score(board, player)
```

Comment:
This looks like some extraneous code that crept in somehow.

I don't know my Python that well, but I have a feeling the way this `for` works is that it only executes once, when the `player` is `O`.  I don't think the second `X` ever results in any code execution.  The `return` statement runs and returns the value of `self.terminal_state_score(board, O)`.  And it turns out that this works because `terminal_state_score` doesn't use the second parameter, `player`.